scripts/fast-reboot-dump.py

```diff
@@ -80,23 +80,45 @@ def get_bridge_port_id_2_port_id(db):
 
     return bridge_port_id_2_port_id
 
-def get_map_port_id_2_iface_name(db):
+def get_lag_name(member_name, app_db):
+    keys = app_db.keys(app_db.APPL_DB, 'LAG_MEMBER_TABLE:PortChannel*')
+    keys = [] if keys is None else keys
+    for key in keys:
+        entry = key.split(":")
```

Comment:
I suggest more verbose way to parse the key:
```suggestion
        _, lag_name, lag_member_name = key.split(":")
```

Reply:
Fixed